src/utils/newRelease.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import FlashMessageHolder from '../components/toolbox/flashMessage/holder';
+import NewReleaseMessage from '../components/newReleaseMessage/newReleaseMessage';
+
+export default {
+  init: () => {
+    const { ipc } = window;
+    if (!ipc) return;
+
+    ipc.on('update:available', (action, { version, releaseNotes }) => {
+      /* istanbul ignore next */
```

Comment:
Why? Add a test or a comment here, please.

Reply:
Added while developing and forgot there 🤦‍♂